care/facility/migrations/0468_alter_asset_asset_class_alter_asset_meta.py

```
@@ -0,0 +1,25 @@
+# Generated by Django 5.1.2 on 2024-11-20 20:53
+
+import care.facility.models.json_schema.asset
+import care.utils.models.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('facility', '0468_alter_facilitypatientstatshistory_unique_together_and_more'),
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)